typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
```

Comment:
There seems to be a circular reference here.  global_vars imports from logger, yet logger imports from global_vars, and tries to use values defined in it.